engine/src/main/java/com/redhat/cloud/notifications/recipients/itservice/pojo/request/ITUserRequest.java

```diff
@@ -8,7 +8,7 @@
     public ITUserRequestBy by;
     public Include include;
 
-    public ITUserRequest(String accountId, boolean adminsOnly) {
+    public ITUserRequest(String accountId, boolean adminsOnly, int pagingStart, int pagingEnd) {
```

Comment:
Can we stick with the IT API names?
```suggestion
    public ITUserRequest(String accountId, boolean adminsOnly, int firstResult, int maxResults) {
```